nautobot/extras/migrations/0041_note.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.2.14 on 2022-07-25 21:50
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import nautobot.core.fields
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("extras", "0040_job__add_is_job_hook_receiver"),
```

Comment:
Note to self: Double check recent PRs haven't caused a conflict here (thinking @gsnider2195's recent work).